style.css

```diff
@@ -0,0 +1,421 @@
+  color: var(--clr-purple);
+  background-color: transparent;
+  letter-spacing: 0.03rem;
+  font-size: 17px;
+}
+
+.card-button:hover {
+  background-color: var(--clr-purple);
+  color: var(--clr-white);
+}
+
+.about-btn:hover {
+  background-color: var(--clr-purple);
+  color: var(--clr-white);
+}
+
+.skill-list li {
+  list-style: none;
+}
+
+.skill-list .p {
+  font-weight: 500;
+  font-size: 20px;
+  color: #000;
+  padding: 1.8rem 0;
+}
+
+.skill-list ul {
+  display: flex;
+  flex-direction: column;
+  padding: 0;
+  margin-top: 2rem;
+}
+
+.hr {
+  border-bottom: 1px solid gray;
+  border-top: 1px solid gray;
+}
+
+.counter {
+  margin-top: 10px;
+  float: right;
+}
+
+.form-group {
+  display: flex;
+  flex-direction: column;
+  background-color: var(--clr-purple);
+  box-sizing: border-box;
+  border: 1px solid var(--clr-purple);
+  border-top-left-radius: 5rem;
+  padding: 6rem 1.8rem;
+  height: auto;
+  box-shadow: -5px -15px 10px -10px rgba(83, 81, 81, 0.15);
+}
+
+.form-group h2 {
+  text-align: center;
+  font-size: 2.5rem;
+  line-height: 52px;
+  color: var(--clr-white);
+  padding-top: 6rem;
+}
+
+.form-input {
+  position: relative;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  box-sizing: border-box;
+  background: var(--clr-white);
+  border: 1px solid #dfe1e6;
+  padding: 1rem;
+  gap: 12px;
+  border-radius: 1rem;
+  margin: 2rem 2rem 0 2rem;
+  box-shadow: 8px 8px 10px -10px rgb(209, 204, 204);
+}
+
+.form-group p {
+  font-size: 1.2rem;
+  line-height: 28px;
+  text-align: center;
+  color: var(--clr-white);
+  font-weight: 400;
+  display: flex;
+  margin: 2rem 0;
+}
+
+.form-group input,
+textarea {
+  background-color: var(--clr-white);
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  padding: 0.8rem 0;
+  margin: 10px 0;
+  width: 100%;
+  color: var(--clr-gray-100);
+}
+
+::placeholder {
+  font-family: 'Poppins', sans-serif;
+  color: var(--clr-gray-100);
+  font-size: 15px;
+  padding-left: 1rem;
+}
+
+.form-bg-img {
+  position: absolute;
+  width: 60%;
+  height: auto;
+  left: 160px;
+}
+
+.card-frame {
+  display: flex;
+  gap: 0.8rem;
+  padding: 0;
+  margin-bottom: 0.5rem;
+}
+
+.error {
+  position: absolute;
+  left: 10rem;
+  bottom: 1rem;
+  color: rgb(255, 0, 0);
+  font-weight: bold;
+}
+
+.card-frame li {
+  font-size: 14px;
+  margin: 5px;
+  color: #7a869a;
+}
+
+.btn {
+  background: var(--clr-white);
+  display: flex;
+  cursor: pointer;
+}
+
+.btn:hover {
+  background-color: var(--clr-purple);
+  color: var(--clr-white);
+  size: 3px;
+}
+
+.card-tags {
+  padding: 0;
+  margin: 1rem 0;
+  list-style: none;
+  display: flex;
+}
+
+/* About Me styles */
+
+.about {
+  box-sizing: border-box;
+  background: var(--clr-white);
+  border: 1px solid #f5f6f8;
+  width: auto;
+  height: max-content;
+  display: flex;
+  flex-direction: column;
+  border-top-right-radius: 5rem;
+  box-shadow: 0 -10px 10px -10px rgba(83, 81, 81, 0.15);
+  margin-bottom: 3rem;
+}
```

Comment:
Also, please check the design for the About Myself section 😉:
-  Kindly be sure to use the corresponding color for the heading (#172B4D).
 
| Figma design  | Your design | 
| ------------- | ------------- | 
| ![image](https://user-images.githubusercontent.com/42799579/205346081-213afc01-5fdc-4a61-8c22-10da69db3d2b.png)  | ![image](https://user-images.githubusercontent.com/42799579/205346028-6e768547-331b-42bb-b698-ee6a98b1ae82.png)  | 

- The same applies to the paragraph:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205346254-42ccef0d-df40-451a-8931-02f0f46db39f.png)  | ![image](https://user-images.githubusercontent.com/42799579/205346216-440e6dfb-511c-4bec-acf7-beb2ea797f43.png)  |

> Note: you can always select any element on Figma, and a window with the details will be shown 😃

- Please check the margin between your elements:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205346515-ff72815c-2c60-4a8d-a4be-e0cd9aa25fb0.png)  | ![image](https://user-images.githubusercontent.com/42799579/205346463-416be1a6-1528-4f2f-b3a5-1d4ca7144129.png)  |

- And finally, the arrow pointing down in the original design & the gap between the icon and the texts:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/205346882-ab10c69b-de45-4dda-8ad6-abea80a9682b.png)  | ![image](https://user-images.githubusercontent.com/42799579/205346924-f4f2586a-d2fd-4a5e-ab49-0e0ed7dd2182.png)  |

Don't forget that we need to check if you are learning how to follow design guidelines 😄. Cheers and keep up the good work!